robottelo/ui/computeresource.py

```diff
@@ -143,9 +143,125 @@ def update(self, name, newname=None, parameter_list=None,
 
     def delete(self, name, really=True):
         """Removes the compute resource entity"""
+        Navigator(self.browser).go_to_compute_resources()
         self.delete_entity(
             name,
             really,
             locators['resource.delete'],
             drop_locator=locators['resource.dropdown'],
         )
+
+    def go_to_compute_resource(self, res_name):
```

Comment:
After going through the code, realized that this is actually the `compute_resource: show_page`. It's like accessing a particular compute_resource in the compute_resource index page.
Renaming it as below:
`s/go_to_compute_resource/access_compute_resource/`